### PR TITLE
Move the slides script out of index.html #2

### PR DESCRIPTION
**Pull request by jenster5**
Fixing issue/Hacktoberfest ---Move the slides script out of index.html #2

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -334,6 +334,7 @@ gulp.task(
       .src(
         [
           './index.html',
+          'main.js',
```
Comment (AhsanAyaz) [inline gulpfile.js]: @jenster5  can you please replace this with the following for consistency? :
```suggestion
          './main.js',

```

File: main.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        link: 'wisdom-perfection-and-crafting-excellence-at-work.html',
+        title: 'Wisdom, Perfection, and crafting Excellence at Work',
+    },
+    {
+        link: 'ui-for-backenders.html',
+        title: 'UI for Backenders',
+    }
+    ];
+    const grid = document.getElementById('talksGrid');
```
Comment (AhsanAyaz) [inline main.js]: @jenster5 this file doesn't need to be formatted properly. Can you please format it?

**Review by AhsanAyaz — CHANGES_REQUESTED** [review]
@jenster5 Good work. Just added a few comments that need to be resolved.

File: gulpfile.js (hunk 1 above)
Comment (jenster5) [reply gulpfile.js]: good catch done

File: main.js (hunk 2 above)
Comment (jenster5) [reply main.js]: applied prettier here

**Review by AhsanAyaz — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**